docs/pages/enroll-resources/desktop-access/getting-started.mdx

```diff
@@ -84,10 +84,23 @@ interactively and select the Teleport certificate that you exported when prompte
 If you want to automate the installation process, you can run the setup program from
 an administrative Command Prompt or PowerShell console with the following command:
 
+<Tabs>
```

Comment:
We don't need tabs for this. Just use `teleport.version` unconditionally.

Reply:
The version are different though depending on which flow they're following. If they pull from community/ent that's 16.4 in this case. Cloud's exec name would be 16.3 version.


![image](https://github.com/user-attachments/assets/f0ab725e-527a-4c63-a9e5-3ed88305b8ed)

Reply:
one uses `teleport.version` and the other tab `cloud.version`.

Reply:
Did you want to take out the above tabs too?

Reply:
got it, please see update, tabs in both sections removed and only using `teleport.version`.